atlp-rwanda/cabal-bn-be
# #181339473 integrate circle ci with readme

### What does this PR do?
Integrate circle-ci badge with readme & adding coverage

### Description of Task to be completed?

- adding circle-ci .yml file and configuring it
- connecting coverage to circle-ci

### How should this be manually tested?

- Clone this repository and open the project in your code editor.
- Check out  [chore/#181339473-intergrate-circle-ci-with-readme](https://github.com/atlp-rwanda/cabal-bn-be/tree/chore/%23181339473-intergrate-circle-ci-with-readme)
- to install all package on your local repo use: **yarn install** 
- do a commit on this branch

### What are the relevant pivotal tracker stories?
[#181339473](https://www.pivotaltracker.com/n/projects/2555280)
### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/95901517/156580624-0aeb623f-b4c2-47bf-9bb0-046c05be79f0.png)
